src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -82,6 +82,11 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        preg_match('/^t.{0,3}:d.{0,4}$/', $input->getFirstArgument(), $matches);
+        if (count($matches)) {
+            trigger_error(sprintf('The use of "%s" command is deprecated since version 2.7 and will be removed in 3.0. Use the "debug:twig" instead.', $input->getFirstArgument()), E_USER_DEPRECATED);
```

Comment:
I suggest using ``twig:debug`` in the message rather than using the shortcut if any.

and I see 2 issues here:

- this code should not be in the bridge command, as the bridge command does not set the BC alias. It should be in the child class in TwigBundle only
- the code should display a nice warning in the output instead of triggering a deprecation warning, so that the message can be seen. People cannot see it in the profiler in the CLI given there is no profiler there. And I doubt people will read their log files when running such commands to see the warning being in it. So they won't see the warning